cmd/minishift/cmd/start.go

```diff
@@ -187,13 +189,21 @@ func runStart(cmd *cobra.Command, args []string) {
 		atexit.Exit(1)
 	}
 
+	automountHostfolders(host.Driver)
+
 	clusterUp(&config, ip)
 	if err := openshift.AddSudoersRoleForUser("developer"); err != nil {
 		glog.Errorln("Error adding developer user to sudoers", err)
 		atexit.Exit(1)
 	}
 }
 
+func automountHostfolders(driver drivers.Driver) {
+	if hostfolder.IsAutoMount() && hostfolder.IsHostfoldersDefined(false) {
```

Comment:
`List`, `Mount`, `Umount` and the `MountHostfolders`. The reasoning is to prevent duplication of code... but I can surely place this inside the function, extract the const, etc.